src/Endpoints/Tasks.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MeiliSearch\Endpoints;
+
+use MeiliSearch\Contracts\Endpoint;
+use MeiliSearch\Exceptions\TimeOutException;
+
+class Tasks extends Endpoint
```

Comment:
Since these do not match the naming used above, I would create to file in `endpoints`
`ClientTasks.php` and `IndexTasks.php`

Reply:
another option would be to be able to do 
```php
$this->tasks->index->getTasks(self::PATH.'/'.$this->uid);

# OR
$this->tasks->client->getTasks(self::PATH.'/'.$this->uid);
```